src/gasman.c

```diff
@@ -324,7 +324,7 @@ void CHANGED_BAG_IMPL(Bag bag) {
 #define SpaceBetweenPointers(a,b) (((UInt)((UInt)(a) - (UInt)(b)))/sizeof(Bag))
 
 #define SizeMptrsArea SpaceBetweenPointers(OldBags, MptrBags)
-#define SizeOldBagsArea SpaceBetweenPointers(YoungBags,OldBags)
+#define SizeOldBagsArea SpaceBetweenPointers(YoungBags, OldBags)
 #define SizeYoungBagsArea SpaceBetweenPointers(AllocBags, YoungBags)
 #define SizeAllocationArea SpaceBetweenPointers(StopBags, AllocBags)
 #define SizeUnavailableArea SpaceBetweenPointers(EndBags, StopBags)
```

Comment:
Yes, unless we want to take this opportunity to remove relics of this feature entirely.